include/zephyr/net/wifi.h

```diff
@@ -118,6 +118,10 @@ const char *wifi_band_txt(enum wifi_frequency_bands band);
 #define WIFI_SAE_PSWD_MAX_LEN 128
 /** MAC address length */
 #define WIFI_MAC_ADDR_LEN 6
+/** Max identity length */
+#define WIFI_IDENTITY_MAX_LEN 64
+/** Max password length */
+#define WIFI_PSWD_MAX_LEN 128
```

Comment:
may be add a prefix to indicate that this is for for Enterprise?

Reply:
updated